desk/lib/channel-utils.hoon

```diff
@@ -739,8 +748,13 @@
         ?(%code %inline-code)  ''
         %ship                  (scot %p p.c)
         %task                  (flatten [%inline q.c]~)
+    ::
         ?(%italics %bold %strike %blockquote)
       (flatten [%inline p.c]~)
+    ::
+        %sect
+      ?~  p.c  '@all'
+      (cat 3 '@' (scot %tas p.c))
```

Comment:
This is not really regarding the backend code, but I was curious whether or not these are going to be rendered differently from nicknames at the client. If not, there is a collision risk when someone cunningly sets their nickname to a role or even 'all'

Reply:
at this time, no they aren't rendered differently. which is also a problem currently as you can set a nickname to match someone else or impersonate a ship. as far as the actual mention mechanism itself, I think we're safe as long as someone doesn't make a role named "all". This would probably trip up the frontend but maybe I can make a special ID for the frontend to prevent this from being an issue. Someone would just have to be careful between choosing all the role vs all.